tests/scripts/thread-cert/Cert_5_3_03_AddressQuery.py

```diff
@@ -85,64 +111,36 @@ def test(self):
         self.nodes[DUT_ROUTER2].start()
         self.nodes[ROUTER3].start()
         self.nodes[MED1].start()
-        self.simulator.go(5)
+        self.simulator.go(10)
 
         self.assertEqual(self.nodes[ROUTER1].get_state(), 'router')
         self.assertEqual(self.nodes[DUT_ROUTER2].get_state(), 'router')
         self.assertEqual(self.nodes[ROUTER3].get_state(), 'router')
         self.assertEqual(self.nodes[MED1].get_state(), 'child')
 
         # 2
-        # Flush the message queue to avoid possible impact on follow-up
-        # verification.
-        dut_messages = self.simulator.get_messages_sent_by(DUT_ROUTER2)
-
         router3_mleid = self.nodes[ROUTER3].get_ip6_address(config.ADDRESS_TYPE.ML_EID)
         self.assertTrue(self.nodes[MED1].ping(router3_mleid))
 
-        # Verify DUT_ROUTER2 sent an Address Query Request to the Realm local
-        # address.
-        dut_messages = self.simulator.get_messages_sent_by(DUT_ROUTER2)
-        msg = dut_messages.next_coap_message('0.02', '/a/aq')
-        command.check_address_query(
-            msg,
-            self.nodes[DUT_ROUTER2],
-            config.REALM_LOCAL_ALL_ROUTERS_ADDRESS,
-        )
-
         # 3
         # Wait the finish of address resolution traffic triggerred by previous
         # ping.
         self.simulator.go(5)
 
-        # Flush the message queue to avoid possible impact on follow-up
-        # verification.
-        dut_messages = self.simulator.get_messages_sent_by(DUT_ROUTER2)
-
         med1_mleid = self.nodes[MED1].get_ip6_address(config.ADDRESS_TYPE.ML_EID)
         self.assertTrue(self.nodes[ROUTER1].ping(med1_mleid))
 
-        # Verify DUT_ROUTER2 responded with an Address Notification.
-        dut_messages = self.simulator.get_messages_sent_by(DUT_ROUTER2)
-        msg = dut_messages.next_coap_message('0.02', '/a/an')
-        command.check_address_notification(msg, self.nodes[DUT_ROUTER2], self.nodes[ROUTER1])
+        self.collect_ipaddrs()
+        self.collect_rlocs()
+        self.collect_rloc16s()
```

Comment:
Any specific reasons to put these between pings? How about moving these after topology is formed?

Reply:
moved to the topology formed